src/include/ipc/header.h

```diff
@@ -152,6 +152,7 @@
 #define SOF_IPC_TRACE_DMA_PARAMS		SOF_CMD_TYPE(0x001)
```

Comment:
@ktrzcinx any idea who sets the initial trace level?

Reply:
`DECLARE_TR_CTX(buffer_tr, SOF_UUID(buffer_uuid), LOG_LEVEL_INFO);`